libs/posthog_client_lite/src/background_loop.rs

```diff
@@ -55,9 +55,16 @@ impl FeatureResolverBackgroundLoop {
                             continue;
                         }
                     };
-                    let feature_store = FeatureStore::new_with_flags(resp.flags);
-                    this.feature_store.store(Arc::new(feature_store));
-                    tracing::info!("Feature flag updated");
+                    let project_id = this.posthog_client.config.project_id.parse::<u64>().ok();
```

Comment:
nit: if it should always be u64, let's set the type of the field in the config to `u64`

Reply:
We already created that item in the pageserver config so it's hard to change now 🤣 I'd rather keep it as-is